metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/RunExecutionAthenaAggregator.java

```diff
@@ -164,6 +176,18 @@ protected String getMaxColumnName() {
         return "max_" + substitutePeriodsForUnderscores(getMetricColumnName());
     }
 
+    protected String get5thPercentileColumnName() {
+        return "percentile05th_" + substitutePeriodsForUnderscores(getMetricColumnName());
+    }
+
+    protected String get95thPercentileColumnName() {
```

Comment:
Would be nice to name this (and related functions) consistently (`getPercentile95thColumnName`).